src/v/pandaproxy/schema_registry/store.h

```diff
@@ -532,6 +532,16 @@ class store {
         return !found;
     }
 
+    void clear() {
+        if (!_schemas.empty()) {
+            _schemas.clear();
+        }
+
+        if (!_subjects.empty()) {
+            _subjects.clear();
+        }
```

Comment:
```suggestion
        _schemas.clear();
        _subjects.clear();
```

Reply:
question, does `compatability_level` need to be reset as well?

Reply:
> question, does `compatability_level` need to be reset as well?

I was wondering the same thing. I'll reset it here just to be safe.